topdown/cache.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/open-policy-agent/opa/util"
 )
 
+type VirtualCache interface {
```

Comment:
A short comment too for this would be helpful now that this functionality is public.